src/components/ItemsList.js

```diff
@@ -73,7 +74,8 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
-  const listHasAtLeastOneItem = shoppingList && shoppingList[0];
+  let listHasAtLeastOneItem = shoppingList && shoppingList[0];
```

Comment:
Just a question for my understanding- what is the reason this has changed from const to let?

Reply:
ooh nothing in particular, I changed it when I wanted to log something to the console that demanded that I reassign the `listHasAtLeastOneItem`, I have updated it now.